Unix/etc/omiserver.conf

```diff
@@ -58,3 +58,5 @@ NoSSLCompression=false
 #NtlmCredsFile=PATH
 # For example
 #NtlmCredsFile=/etc/opt/omi/.creds/ntlm
+nonroot=false
+service=test1
```

Comment:
Should defaults be added to the "real" omiserver.conf file shipping to customers on a kit install?